materialrangebar/src/main/java/com/appyvet/materialrangebar/Bar.java

```diff
@@ -161,6 +189,85 @@ public Bar(Context ctx,
+     * @param tickCount        the number of ticks on the bar
+     * @param tickHeight       the height of each tick
+     * @param tickColor        the color of each tick
+     * @param barWeight        the weight of the bar
+     * @param barColor         the color of the bar
+     * @param isBarRounded     if the bar has rounded edges or not
+     * @param tickLabelColor   the color of each tick's label
+     * @param tickTopLabels    the top label of each tick
+     * @param tickBottomLabels the top label of each tick
+     */
+    public Bar(Context ctx,
+               float x,
+               float y,
+               float length,
+               int tickCount,
+               float tickHeight,
+               int tickColor,
+               float barWeight,
+               int barColor,
+               boolean isBarRounded,
+               int tickLabelColor,
+               int tickLabelSelectedColor,
+               CharSequence[] tickTopLabels,
+               CharSequence[] tickBottomLabels,
+               String tickDefaultLabel,
+               float tickLabelSize) {
+        this(ctx, x, y, length, tickCount, tickHeight, barWeight, barColor, isBarRounded, tickLabelColor, tickLabelSelectedColor, tickTopLabels, tickBottomLabels, tickDefaultLabel, tickLabelSize);
+        mTickPaint.setColor(tickColor);
+    }
+
+    /**
+     * Bar constructor
+     *
+     * @param ctx              the context
+     * @param x                the start x co-ordinate
+     * @param y                the y co-ordinate
+     * @param length           the length of the bar in px
+     * @param tickCount        the number of ticks on the bar
+     * @param tickHeight       the height of each tick
+     * @param tickColors       the colors of each tick
+     * @param barWeight        the weight of the bar
+     * @param barColor         the color of the bar
+     * @param isBarRounded     if the bar has rounded edges or not
+     * @param tickLabelColor   the color of each tick's label
+     * @param tickTopLabels    the top label of each tick
+     * @param tickBottomLabels the top label of each tick
+     */
+    public Bar(Context ctx,
+               float x,
+               float y,
+               float length,
+               int tickCount,
+               float tickHeight,
+               List<Integer> tickColors,
+               float barWeight,
+               int barColor,
+               boolean isBarRounded,
+               int tickLabelColor,
+               int tickLabelSelectedColor,
+               CharSequence[] tickTopLabels,
+               CharSequence[] tickBottomLabels,
+               String tickDefaultLabel,
+               float tickLabelSize) {
+
+        this(ctx, x, y, length, tickCount, tickHeight, barWeight, barColor, isBarRounded, tickLabelColor, tickLabelSelectedColor, tickTopLabels, tickBottomLabels, tickDefaultLabel, tickLabelSize);
+
+        if (null != tickColors) {
```

Comment:
Please try to use non inverted conditions `if(null != tickColors)` can be replaced with 
`if(tickColors != null)`

Reply:
Done